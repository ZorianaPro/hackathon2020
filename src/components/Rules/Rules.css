.Rules {
    margin: 0 auto;
    padding: 0 10% 5% 10%;
    background-color:  var(--colorPapayesWhip);
}

.Rules .Rules-Title {
    font-size: var(--fontSize48);
    font-weight: var(--fontWeightPoppinsBold);
    font-stretch: normal;
    font-style: italic;
    line-height: var(--lineHeight);
    letter-spacing: normal;
    text-align: center;
    color: var(--colorSpaceCadet);
}

.Rules-List {
    margin: 0;
    padding: 0;
    list-style: none;
}

.Rules-List .Rules-Item h2::before {
    content: '\01F31F';
    font-size: 1.25em;
    display: inline;
    font-weight: 100;
    padding-right: 0.5em;
  }