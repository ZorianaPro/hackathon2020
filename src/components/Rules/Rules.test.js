import React from 'react';
import { shallow, mount } from 'enzyme';
import Rules from '../Rules';

describe('Rules', () => {
  let rules;

  beforeEach(() => {
    rules = mount(<Rules />);
  });

  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <Rules />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(rules).toMatchSnapshot();
  });
});
