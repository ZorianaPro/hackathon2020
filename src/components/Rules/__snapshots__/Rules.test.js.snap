// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules renders as expected 1`] = `
<Rules>
  <div
    className="Rules"
  >
    <ul
      className="Rules-List"
    >
      <li
        className="Rules-Item"
      >
        <h2>
          FRESH CODE BUT BE PREPARED
        </h2>
        <p>
          No matter the project, you should start coding together and not start hacking beforehand. What you can do however, is sit down with your respective team at least a few days before the hackathon and discuss what you need for your initial setup, what you intend to do exactly, how the tasks will be divided between the team members and what you will need in terms of hard-/software.
        </p>
      </li>
      <li
        className="Rules-Item"
      >
        <h2>
          TEAM SETUP
        </h2>
        <p>
          Teams should consist of 2-3 members. You can pick your project and team partners as you like, but be mindful to mix with people you normally don't work with. That's the most important rule. Also think about working on a topic you normally don't work on, this is what the hackathon is for!
        </p>
      </li>
      <li
        className="Rules-Item"
      >
        <h2>
          DEMO YOUR WORK
        </h2>
        <p>
          At the end of the Hackathon, every team will have 3-5min to make a DEMO, and 3-5min for Q&A from the crowd.
        </p>
      </li>
      <li
        className="Rules-Item"
      >
        <h2>
          WINNING
        </h2>
        <p>
          After all demos, everyone will vote for their 3 favourite teams and rank them. The link to the voting will be shared on the day of the event.
        </p>
      </li>
    </ul>
  </div>
</Rules>
`;
